src/SelfAssessment/SelfAssessmentIntro.tsx

```diff
@@ -30,6 +31,12 @@ const SelfAssessmentIntro: FunctionComponent = () => {
     navigation.navigate(SelfAssessmentStackScreens.HowAreYouFeeling)
   }
 
+  const handleOnPressCDCLink = () => {
+    const cdcSelfCheckerURL =
+      "https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/coronavirus-self-checker.html"
```

Comment:
Should this be put in the env as a link?

Reply:
I don't think so? At this point it's constant for anyone using the self-checker. Thoughts?

Reply:
Merging for now but we can add this to the env later if needed.